vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/dif/InputEvaluator.kt

```diff
@@ -0,0 +1,147 @@
+package at.asitplus.wallet.lib.data.dif
+
+import at.asitplus.KmmResult
+import at.asitplus.jsonpath.JsonPath
+import at.asitplus.jsonpath.core.NodeListEntry
+import io.github.aakira.napier.Napier
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonNull
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.booleanOrNull
+import kotlinx.serialization.json.doubleOrNull
+import kotlinx.serialization.json.longOrNull
+
+/*
```

Comment:
Please use `/**` and also move the comment in `//` into here

Reply:
fixed in 3e4c8cd54334e7b7ed0bb56817ae64d8ed0d8d0d